server/report_builder/README.md

```diff
@@ -410,3 +410,55 @@ Where value is the translated value of 'label.value' in our `common.json` transl
 The "label" of the ui schema controls the text displayed in the front end report filtering modal.
 
 This function could also be used on any other serialised json value such as patient json schema.
+
```

Comment:
This is a bit rough and ready - but I wanted to make sure it was at least captured. 

Will soon be restructured in [this issue](https://github.com/msupply-foundation/open-msupply/issues/5037)